lib/Mojolicious/Command/coverage.pm

```diff
@@ -0,0 +1,134 @@
+package Mojolicious::Command::coverage;
+use strict;
+use warnings FATAL => 'all';
+use Mojo::Base 'Mojolicious::Command', -signatures;
+use Mojo::Util qw(getopt);
+
+our $VERSION = "0.0.0"; # Do not update manually
+
+# Short description
+has description => 'Start you Mojo app in coverage mode';
+
+# Usage message from SYNOPSIS
+has usage => sub($self) {$self->extract_usage};
+
+
+sub run($self, @args) {
+    my $deanonConfig = "";
+    my $coverageConfig = "";
+    my $n_args = 0;
+    my @orig_args = @args;
+
+    getopt \@args,['pass_through'],
+        'd|deanon-args=s' => sub {
+            $n_args+=2;
+            $deanonConfig = $_[1]
+        },
+        'c|cover-args=s'  => sub {
+            $n_args+=2;
+            $coverageConfig = $_[1]
+        };
+
+    if ($deanonConfig eq "" && $self->app->can('deanonymizeConfig')) {
+        $deanonConfig = $self->app->deanonymizeConfig;
+        print "Command::coverage: Has deanonymize config \n";
+    }
+
+    if ($coverageConfig eq "" && $self->app->can('coverageConfig')) {
+        $coverageConfig = $self->app->coverageConfig;
+        print "Command::coverage: Has coverage config \n";
+    }
+
+    # if there is no custom config, we fallback to default
+    $deanonConfig = ref $self->app if $deanonConfig eq "";
+    $coverageConfig = "-ignore,t/,-coverage,statement,branch,condition,path,subroutine" if $coverageConfig eq "";
+
+    my $command_line = "";
+    my $forwarded_args = join " ", @orig_args[$n_args..$#orig_args];
+    if ($coverageConfig ne "0"){
+        $command_line = "perl -MDevel::Cover=$coverageConfig -MDevel::Deanonymize=$deanonConfig $0 daemon $forwarded_args";
+    } else {
+        $command_line = "perl -MDevel::Cover=$coverageConfig $0 daemon $forwarded_args";
+    }
+
+    print "Command::coverage: Starting application with: `$command_line`";
+    exec $command_line;
```

Comment:
aus sicherheitsgruenden wuerde ich die argumente in einem array uebergeben, dann kommt sicher keine shell zum einsatz.